src/style.css

```diff
@@ -1 +1,35 @@
 /* add styles here */
+:root{
+  --main-images: (./images/star-active.svg)
+}
```

Comment:
The `--main-images` custom property is defined but never used. If it's intended for use, make sure to implement it where necessary. Otherwise, consider removing unused code to keep the stylesheet clean.